packages/auth/src/platform_browser/providers/phone.test.ts

```diff
@@ -18,19 +18,31 @@
 import { expect } from 'chai';
 import * as sinon from 'sinon';
 
-import { mockEndpoint } from '../../../test/helpers/api/helper';
+import {
+  mockEndpoint,
+  mockEndpointWithParams
+} from '../../../test/helpers/api/helper';
 import { testAuth, TestAuth } from '../../../test/helpers/mock_auth';
 import * as fetch from '../../../test/helpers/mock_fetch';
-import { Endpoint } from '../../api';
+import {
+  Endpoint,
+  RecaptchaClientType,
+  RecaptchaVersion,
+  RecaptchaAuthProvider,
+  EnforcementState
+} from '../../api';
 import { RecaptchaVerifier } from '../../platform_browser/recaptcha/recaptcha_verifier';
 import { PhoneAuthProvider } from './phone';
+import { FAKE_TOKEN } from '../recaptcha/recaptcha_enterprise_verifier';
+import { MockGreCAPTCHATopLevel } from '../recaptcha/recaptcha_mock';
 
 describe('platform_browser/providers/phone', () => {
   let auth: TestAuth;
 
   beforeEach(async () => {
     fetch.setUp();
     auth = await testAuth();
+    auth.settings.appVerificationDisabledForTesting = false;
```

Comment:
Will the tests fail if it's set to true?

Reply:
Yes, tests with Recaptcha Enterprise enabled will fail if appVerificationDisabledForTesting is set to true.

(I mistakenly removed the test in the previous commit, just added it back)